composer.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "oomphinc/drupal-scaffold",
+    "description": "A Drupal 8 scaffold project",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nathan Dentzau",
+            "email": "nathan.dentzau@gmail.com"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    ],
+    "require": {
+        "bower-asset/bootstrap": "^4.0",
+        "composer/installers": "^1.5",
+        "cweagans/composer-patches": "^1.6",
+        "drupal-composer/drupal-scaffold": "^2.4",
+        "drupal-composer/preserve-paths": "^0.1.4",
+        "drupal/admin_toolbar": "^1.23",
+        "drupal/coder": "^8.2",
```

Comment:
What's coder do? Supply the phpcs standards and stuff?

Reply:
Yup it's the Drupal PHPCS coding standard.